src/blueberry-muffins.md

```diff
@@ -0,0 +1,55 @@
+# Blueberry Muffins
+
+![blueberry-muffins](/data/pix/blueberry-muffins.webp)
```

Comment:
```suggestion
![blueberry-muffins](pix/blueberry-muffins.webp)
```

The structure of the website is a little different than the repo, so the correct path for the images is without `/data/`.

Reply:
I see, makes sense!